lib/slack-ruby-bot-server/config.rb

```diff
@@ -3,11 +3,30 @@ module Config
     extend self
 
     attr_accessor :server_class
+    attr_accessor :database_adapter
 
     def reset!
       self.server_class = SlackRubyBotServer::Server
     end
 
+    def database_adapter
```

Comment:
You could move this into an initializer. We call `reset!`, so maybe just set it there?